www/src/components/guidelines/color/swatch.js

```diff
@@ -1,132 +1,122 @@
 /** @jsx jsx */
 import { jsx } from "theme-ui"
-import React from "react"
+import React, { useState } from "react"
 import CopyToClipboard from "react-copy-to-clipboard"
 
 import { Box } from "../system"
```

Comment:
Would be nice to get rid of this at some point.

Reply:
@fk Yes, I will convert this also to theme-ui, then we will get rid of React import here. I did not wanted to touch two separate concerns in 1 PR. 
Thank you very much.

Reply:
👍 🙏

Reply:
@fk I think we cannot remove React import because we are using useState hook. But, I have converted this component to theme-ui, there were some css rules which were repeated, I have removed them also. I will raise the PR in few minutes.

Reply:
Looking already! 🙌